### PR TITLE
feat(Pokémon-types):add Pokémon types and the switch depending in the Pokémon type

### LINKED ISSUE #48: Have types be a specific color
**Is your feature request related to a problem? Please describe.**
All Pokémon types container are yellow and if a Pokémon has more that one type the div cant hold both inside

**Describe the solution you'd like**
Every Pokémon type be a specific color depending on what type it is. If a Pokémon has two types they should be separated and colored differently.

### LINKED ISSUE #49: No Types on Pokémon vs screen
**Is your feature request related to a problem? Please describe.**
There are no types on the Pokémon vs screen for both Pokémon

**Describe the solution you'd like**
To have both Pokémon show their types

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. Changed the types to have a background specific to their color
2. have the types show up for both Pokémon on the vs screen

## Purpose
to show the types of all the Pokémon on the screen

## Approach
Show the enemy Pokémon type and set a class based on the type name and give each one a unique color

Closes #49 
Closes #48

### CODE REVIEW

**Review by BlackCubes — APPROVED** [review]
Looks SO GOOD!

**Review by Wolftownn — APPROVED** [review]
Looking so good!

File: pokemon-app/src/styles/Types.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    background-color: coral;
+}
+.fairy{
+    background-color: pink;
+}
+.dark{
+    background-color: darkslategrey;
+    color: white;
+}
+.toad{
```
Comment (krferrell) [inline pokemon-app/src/styles/Types.css:71]: ooo what is toad?
Excited to see.

**Review by krferrell — APPROVED** [review]
Everything is looking solid to me.

File: pokemon-app/src/styles/Types.css (hunk 1 above)
Comment (Lap343) [reply pokemon-app/src/styles/Types.css:71]: lol shhhhhhh

**Pull request merged**